tests/rptest/clients/kubectl.py

```diff
@@ -321,7 +321,14 @@ def __init__(self, kubectl: KubectlTool, node_name: str) -> None:
         self.logger = self.kubectl._redpanda.logger
         self.current_context = self.kubectl.cmd(
             f"config current-context").decode().strip()
-        self.pod_name = f"{node_name}-priviledged-shell"
+        # Make sure that name is not longer that 63 chars
+        # The Pod "gke-redpanda-co9uuq78jo-redpanda-6a66-fcfacc41-65mz-priviledged-shell" is invalid: metadata.labels:
+        # Invalid value: "gke-redpanda-co9uuq78jo-redpanda-6a66-fcfacc41-65mz-priviledged-shell": must be no more than 63 characters
+        self.pod_name = f"{node_name}-pshell"
+        if len(self.pod_name) > 63:
+            # Assume that our added chars broke the limit
+            # Cut them to fit
+            self.pod_name = self.pod_name[:63]
```

Comment:
This is just a pod name we make up right? So truncating it isn't really an issue.

Reply:
Yes,it is just a pod name to find it if someone really wants  to.